client/editSchedule.js

```diff
@@ -76,6 +75,46 @@ function getRoundsWithSchedule(competitionId) {
   });
   return rounds;
 }
+
+function getScheduledRounds(competitionId) {
+  var rounds = Rounds.find({
+    competitionId: competitionId,
+    startMinutes: {$exists: true}
+  }, {
+    fields: {
+      eventCode: 1,
+      roundCode: 1,
+
+      startMinutes: 1,
+      durationMinutes: 1,
+      nthDay: 1,
+      title: 1,
+    }
+  }).fetch();
+  _.each(rounds, function(round) {
+    extendRound(round, competitionId);
+  });
+  return rounds;
+}
+function getUnscheduledRounds(competitionId) {
+  var rounds = Rounds.find({
+    competitionId: competitionId,
+    startMinutes: {$exists: false}
+  }, {
+    fields: {
+      eventCode: 1,
+      roundCode: 1,
+
+      durationMinutes: 1,
+      nthDay: 1,
+      title: 1,
+    }
+  }).fetch();
+  _.each(rounds, function(round) {
+    extendRound(round, competitionId);
+  });
+  return rounds;
+}
 function extendRound(round, competitionId) {
```

Comment:
Do we still need this? It seems like it was meant to get unscheduled rounds to show up on the calendar nicely, but now they don't need to show up on the calendar anymore =).